app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TweetsController < ApplicationController
+   get '/tweets' do
+      logged_in? ? (erb :'tweets/tweets') : (redirect '/login')
+   end
+
+   get '/tweets/new' do
+      logged_in? ? (erb :'tweets/create_tweet') : (redirect '/login')
+   end
+
+   post '/tweets/new' do
+      if params[:content].empty?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.